Miscellaneous/docker/full_stack_example/config/logstash/logstash.conf

```diff
@@ -0,0 +1,40 @@
+input {
+  tcp {
+    port => 5000
+  }
+}
+
+filter {
+  grok {
+    match => {
+      "message" => '%{IPORHOST:clientip} %{USER:ident} %{USER:auth} \[%{HTTPDATE:timestamp}\] "%{WORD:verb} %{DATA:request} HTTP/%{NUMBER:httpversion}" %{NUMBER:response:int} (?:-|%{NUMBER:bytes:int}) %{QS:referrer} %{QS:agent}'
```

Comment:
Is this an Apache `combined` log? Is there not an existing Grok pattern for that?